infra/hooks/load_azd_env.sh

```diff
@@ -0,0 +1,10 @@
+ #!/bin/sh
+
+echo "Loading azd .env file from current environment..."
```

Comment:
You need to run chmod +x filename.sh on every sh file that you commit.

Reply:
Run this in infra/hooks

`find . -maxdepth 1 -type f -name "*.sh" -exec chmod +x {} \;`